packages/salesforcedx-vscode-apex-debugger/test/unit/core/sessionService.test.ts

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { SfdxCommandBuilder } from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+import { SessionService } from '../../../src/core/sessionService';
+import { CommandOutput } from '../../../src/utils/commandOutput';
+
+describe('Debugger session service', () => {
+  let service: SessionService;
+  const cmdWithArgSpy = sinon.spy(SfdxCommandBuilder.prototype, 'withArg');
+  const cmdWithFlagSpy = sinon.spy(SfdxCommandBuilder.prototype, 'withFlag');
+  const cmdBuildSpy = sinon.spy(SfdxCommandBuilder.prototype, 'build');
+
+  beforeEach(() => {
+    service = new SessionService();
+  });
+
+  describe('Helpers', () => {
+    it('Should detect an Apex Debugger session ID', () => {
```

Comment:
Maybe change the it to be more descriptive like 'Should detect an Apex Debugger session ID by key prefix'?